smartcosmos-framework/pom.xml

```diff
@@ -45,6 +45,10 @@
             <artifactId>spring-cloud-starter-oauth2</artifactId>
         </dependency>
         <dependency>
+            <groupId>org.springframework.security</groupId>
+            <artifactId>spring-security-test</artifactId>
```

Comment:
Okay, I'm going to create a new library for testing stuff, but leave the existing classes where they are.